tensorflow/python/keras/saving/save.py

```diff
@@ -44,6 +44,24 @@
 _KERAS_SAVED_MODEL_STILL_EXPERIMENTAL = True
 
 
+def _could_write_as_hdf5(filepath) -> bool:
+  """detect if `filepath` is a h5py.File object or its extension belongs to HDF5 extensions
```

Comment:
nit: capitalize detect, and this should end with a period. 

Also, include args + returns

Reply:
Thanks, I will update this PR later!